test/functional/settings_test.rb

```diff
@@ -0,0 +1,39 @@
+require File.join(File.dirname(__FILE__), '..', 'test_helper')
+
+describe 'Settings' do
+  let(:setting) do
+  {
+    :name => 'content_action_accept_timeout',
+    :full_name => 'Accept action timeout',
+    :value => 20,
+    :description => 'Time in seconds to wait for a Host to pickup a remote action'
+
+  }
+  end
+
+  describe 'list command' do
+    before do
+      @cmd = %w(settings list)
+    end
+
+    params = []
+
+    it 'lists all settings' do
+      api_expects(:settings, :index, 'List').with_params(
+        'page' => 1, 'per_page' => 1000 
+        ).returns(index_response([setting]))
+
+
+      output = IndexMatcher.new([
+        ['NAME', 'FULL NAME', 'VALUE', 'DESCRIPTION'],
+        ['content_action_accept_timeout', 'Accept action timeout', '20', ]
```

Comment:
Was not matching for the description intentional? You can make the test description shorter if you don't want to have long strings here.